.github/workflows/codespell.yml

```
@@ -0,0 +1,22 @@
+---
+name: Codespell
```

The three dashes (`---`) used to indicate the start of a YAML document should not be mandatory.
```suggestion
name: Codespell
```